Dijkstra's_Algorithm/Dijkstra's_Algorithm.cpp

```diff
@@ -0,0 +1,131 @@
+/*
+
+	Dijkstra's algorithm (or Dijkstra's Shortest Path First algorithm, SPF algorithm) is an algorithm for finding the 
+	shortest paths between nodes in a weighted graph, which may represent, for example, mobile or road networks.
+
+	It can be implemented in C++ using  :
+	1) Adjacency matrix representation
+	2) Adjacency List representation (using vector STL of C++)
+	3) Set STL of C++ 
+	4) PriorityQueue STL of C++
+
+	This code is implementation	of Dijkstra's algorithm using Adjacency list representation for Undirected weighted graph.
+
+	When to use Dijkstra's algorithm and Floyd Warshall algorithm ?
+	-> Dijkstra's algorithm is for single source and multiple destinations .
+	-> Floyd Warshall algorithm is for multiple sources and multiple destinations .
+
+*/
+
+
+#include<bits/stdc++.h>
+using namespace std;
+#define MAXN 1111
+#define INF 999999999
+vector< pair<int, int> >v[MAXN];
+
+vector<int> djikstras(int source, int n) {
+	vector<int> dist(n, INF);
+	set< pair<int, int> > queue;
+	vector<bool> visited(n, false);
+	dist[source] = 0;
+	visited[source] = true;
+	queue.insert(make_pair(dist[source], source));
+
+	while(!queue.empty()) {
+		pair<int, int> front_p = *(queue.begin());
+		queue.erase(queue.begin());
+		int cur_dist = front_p.first;
+		int node = front_p.second;
+
+		for(int i=0; i<v[node].size(); i++) {
+			int to = v[node][i].first;
+			int weight = v[node][i].second;
+			if(dist[to] > cur_dist + weight) {
+				if(queue.find(make_pair(dist[to], to)) != queue.end()) {
+					queue.erase(make_pair(dist[to], to));
+				}
+				dist[to] = cur_dist + weight;
+				queue.insert(make_pair(dist[to], to));
+			}
+		}	
+	}
+	return dist;
+}
+
+int main() {
+	cout<<"\nEnter number of vertices\n";
+	int n; cin >> n;
```

Comment:
don;t use single letter word...please give it a sensible name stating it's purpose